api/utils/keys/policy.go

```diff
@@ -72,6 +72,10 @@ func NewPrivateKeyPolicyError(p PrivateKeyPolicy) error {
 // ParsePrivateKeyPolicyError checks if the given error is a private key policy
 // error and returns the contained unmet PrivateKeyPolicy.
 func ParsePrivateKeyPolicyError(err error) (PrivateKeyPolicy, error) {
+	if err == nil {
+		return "", trace.BadParameter("provided error is not a key policy error")
```

Comment:
Is it possible to make this message a bit clearer? I think it might be easy to read this message and think an err that is non nil was passed in.

Reply:
I've added the original error to the error message

Reply:
I've added the error to the error mesage